tbtadm/controller.cpp

```diff
@@ -809,3 +824,15 @@ void tbtadm::Controller::removeAll()
     fs::remove_all(acltree);
     m_out << count << " entries removed\n";
 }
+
+bool tbtadm::Controller::sysfsDeviceExists()
```

Comment:
No need to make this a member function.
It can be added to the anonymous namespace like the other helpers we have in this file (line 240).